cves/CVE-2018-8037.yml

```diff
@@ -216,4 +231,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The coding mistake was not accounting for the undefined edge case where an async
+    request was completed at the same time as an async timeout.
+    The fixes are proper and account for async behavior.
+    It is interesting that most of the code for this bug/bug fix and the surrounding code
+    is written by one man. He must of missed this condition. Making sure that other async code that exists
+    in tomcat covers the async race condition is crucial to prevent similar bugs in the future.
```

Comment:
Might this issue also be a privacy concern since people are seeing messages intended for others?

Reply:
Agree with matt's comment here, this is a large privacy concern for users as they could see a response not intended by the system.